src/style.css

```diff
@@ -2,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. In your case, `.body` and `.form` look like tag selectors, but they are actually class selectors. It's a bit confusing and it's better to avoid such class names.